seatunnel-connectors-v2/connector-clickhouse/src/main/java/org/apache/seatunnel/connectors/seatunnel/clickhouse/sink/client/ClickhouseSinkWriter.java

```diff
@@ -54,6 +59,13 @@
     private final ShardRouter shardRouter;
     private final transient ClickhouseProxy proxy;
     private final Map<Shard, ClickhouseBatchStatement> statementMap;
+    private transient ScheduledExecutorService scheduler;
+    private transient ScheduledFuture<?> scheduledFuture;
+    private transient JdbcBatchStatementExecutor clickHouseStatement;
+    private transient ClickhouseBatchStatement statement;
+    private transient IntHolder sizeHolder;
+    // Whether pre-initialization is required
+    private transient boolean isOpen;
```

Comment:
This field is not necessary

Reply:
Can the boss point out the number of rows

Reply:
This is a historical commentary Please check the latest